soc/xtensa/intel_adsp/ace_v1x/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
I'll let @nashif  and @mwasko comment, but these could be also folder to the first patch.